src/globus_cli/commands/flows/list.py

```diff
@@ -31,6 +38,7 @@
     "--filter-role",
     type=click.Choice(ROLE_TYPES),
     help="Filter results by the flow's role type associated with the caller",
+    multiple=True,
 )
```

Comment:
This creates a multiple-use option whose resulting parameter name is `filter_role`.
I think it would be preferable to update to
```python
@click.option(
    "--filter-role",
    "filter_roles"
    ...
)
```

Which sets the destination name to `filter_roles`. That way, we don't have a singular/plural discrepancy at the usage sites.